src/ducks/settings/AccountsSettings.jsx

```diff
@@ -62,21 +74,33 @@ const renderAccount = account => (
   <AccountLine account={account} key={account._id} />
 )
 
-const AccountsTable = ({ accounts, t }) => (
-  <Table className={styles.AcnsStg__accounts}>
-    <thead>
-      <tr>
-        <th className={styles.AcnsStg__libelle}>{t('Accounts.label')}</th>
-        <th className={styles.AcnsStg__bank}>{t('Accounts.bank')}</th>
-        <th className={styles.AcnsStg__number}>{t('Accounts.account')}</th>
-        <th className={styles.AcnsStg__type}>{t('Accounts.type')}</th>
-        <th className={styles.AcnsStg__shared}>{t('Accounts.shared')}</th>
-        <th className={styles.AcnsStg__actions} />
-      </tr>
-    </thead>
-    <tbody>{accounts.map(renderAccount)}</tbody>
-  </Table>
-)
+const AccountsTable = ({ accounts, t }) => {
+  const showAccountsOwners = flag('settings.show-accounts-owners')
+
+  return (
+    <Table className={styles.AcnsStg__accounts}>
+      <thead>
+        <tr>
+          <th className={styles.AcnsStg__libelle}>{t('Accounts.label')}</th>
+          <th className={styles.AcnsStg__bank}>{t('Accounts.bank')}</th>
+          <th className={styles.AcnsStg__number}>{t('Accounts.account')}</th>
+          <th className={styles.AcnsStg__type}>{t('Accounts.type')}</th>
+          <th
+            className={
+              showAccountsOwners
+                ? styles.AcnsStg__owner
+                : styles.AcnsStg__shared
+            }
+          >
+            {t(`Accounts.${showAccountsOwners ? 'owner' : 'shared'}`)}
```

Comment:
Nit: if the conditional was on the whole string, it would make it more grep friendly.